deploy/packages/bbp-packages.yaml

```diff
@@ -35,7 +35,7 @@ packages:
       - py-basalt@0.2.4
       - py-entity-management@1.0.16
       - py-jupyter-notebook@4.2.3
-      - brion@3.0.0 +python
+      - brion@3.0.0 +python ^boost@1.65.0
```

Comment:
Sigh. Any details why this has to be?

Reply:
for the time being I want to see if python bindings will be properly built, which is not the case with 1.68.